docs/packaging-applications/build-servers/tfs-azure-devops/index.md

```diff
@@ -1,24 +1,23 @@
 ---
-title: Azure DevOps & Team Foundation Server
-description: Octopus Deploy integrates with Azure DevOps and TFS to provide for a full automated build and deployment pipeline.
+title: Azure DevOps
```

Comment:
I propose keeping "Azure DevOps & Team Foundation Server" since TFS could be used by some customers.

Reply:
@jbristowe So are you saying to just include it in the title or also in the content?

Reply:
we added a banner to some pages